package oopj201221;
import java.util.*;
public class ExceptionHandling{
public static void main(String[] args) {
{
   Scanner sc=new Scanner(System.in);{
   int x=0;
     {
	   System.out.println("---Program to preform division---\n");
       System.out.println("Enter the number 1");
       int num1=sc.nextInt();
       try 
         {
	     System.out.println("entrer the number 2");
         int num2=sc.nextInt();
         System.out.println(num1/num2);}
         catch(ArithmeticException e) {
         System.out.println("Do not divide by zero!!!");
         }
           finally 
            {
      }      System.out.println("End of Operations");
}
   }


 }
}
}
